scripts.js

```diff
@@ -50,6 +52,104 @@ menuContactUs.addEventListener('click', () => {
+    btnimg2: './pictures/TimeAndIcons/github.png',
+  },
+];
+
+if (window.matchMedia('(max-width: 768px)').matches) {
+  document.getElementById('Portfolio-menu').innerHTML = arr1.map((items) => `<div class="card-works">  
+                <h2 class="card-works-title" id="card-works-title">${items.popuptitle}</h2>
+                <p class="card-works-description" id="card-works-description">${items.popupdescription}</p>
+                <ul class="card-works-tech">
+                  <li class="card-works-tech-items">${items.popuptec[0]}</li>
+                  <li class="card-works-tech-items">${items.popuptec[1]}/li>
+                  <li class="card-works-tech-items">${items.popuptec[2]}</li>
+                  <li class="card-works-tech-items">${items.popuptec[3]}</li>
+                </ul>
+                <button class="card-works-button project-2" type="button">
+                  <a href="#popup">See Project</a>
+                </button>
+              </div>`).join('');
+} else {
+  document.getElementById('Portfolio-menu').innerHTML = arr1.map((items) => `<div class="card-works-alt background-2">
+                  <h2 class="card-works-title title-alt">${items.popuptitle}</h2>
+                  <p class="card-works-description description-alt">A${items.popupdescription}</p>
+                  <ul class="card-works-tech tech-alt">
+                    <li class="card-works-tech-items tech-items-alt">${items.popuptec[0]}</li>
+                    <li class="card-works-tech-items tech-items-alt">${items.popuptec[1]}</li>
+                    <li class="card-works-tech-items tech-items-alt">${items.popuptec[2]}</li>
+                    <li class="card-works-tech-items tech-items-alt">${items.popuptec[3]}</li>
+                  </ul>
+                </div>`).join('');
+}
+
+// ---------------------------- OPEN POPUP WINDOW SECTION ----------------------------------
+
+const proj1 = {
+  title: 'Multi Post Stories',
+  img: './images/popup-img.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry´s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+};
+
+const btn1 = document.querySelector('.project-1');
+const popupOpen = document.querySelector('.popup');
+
+btn1.addEventListener('click', () => {
+  popupOpen.classList.toggle('show-popup');
+  document.getElementById('popup-title').innerHTML = proj1.title;
+  document.getElementById('popup-img').src = proj1.img;
+  document.getElementById('popup-description').innerHTML = proj1.description;
+  document.getElementById('popup-live').href = 'https://flunavgt.github.io/Federico-s_Portfolio/';
+  document.getElementById('popup-source').href = 'https://github.com/Camilovelag/MV_Portfolio';
+});
```

Comment:
- [ ] I noticed that you forgot to add the `technologies section` in your popup window, so please add them to look similar to your Figma design, also the images in `See Live` and `See Source` should be replaced 👌. remember According to the requirements You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). 💯

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/180339922-90b871de-04bb-427b-bbd9-f1307b309df9.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/180339904-b3d663fd-ad11-4eac-b372-8ddead6da107.PNG" width="500" /> |

Reply:
![image](https://user-images.githubusercontent.com/106117541/180568705-589928ef-12e2-4fa5-8107-2b8e161cd33d.png)